src/style.scss

```diff
@@ -17,6 +17,8 @@
 // Components
 // ==========================================================================
 
-th.sortable {
-	cursor: pointer;
+th[data-vl-sortable] {
+    a {
+        cursor: pointer;
+    }
```

Comment:
is da nodig op nen a zeggen dat de cursor pointer moet zijn?

Reply:
Ja, deze was nog niet gemeld, maar had ik nog gezien: als het zonder de a tag was, zou heel de th (die groter is in dit geval) cursor pointer hebben, maar enkel de a is clickable. Beetje vreemd om de pointer te hebben maar dat het niet clickable is, niet?

Reply:
Ik denk ofwel de th clickable en dan is deze css fout, ofwel enkel de a clickable en dan is deze overbodig?

Reply:
conclusie: enkel a moet clickable zijn, en deze scss is voorlopig noodzakelijk voor de pointer, maar wordt opgelost in UIG-608

Reply:
Is het nodig om deze stijl toe te voegen aan de a tag? Is die niet pointer by default?

Reply:
Ja, omdat er geen `href` is gedefinieerd. Alternatief zou zijn om `href="javascript:;"` toe te voegen.